dev/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "require": {
```

Comment:
Also, if possible pls add name, type, description, etc fields?

Reply:
I've added "type" and "description", but left off "name" as this implies it to be a [published composer package](https://getcomposer.org/doc/04-schema.md#name), which it isn't